LEAF_Request_Portal/api/RESTfulResponse.php

```diff
@@ -91,8 +91,22 @@ public function output($out = '')
         $format = isset($_GET['format']) ? $_GET['format'] : '';
         switch ($format) {
             case 'json':
+            case 'jsoncompatmode':
             default:
                 header('Content-type: application/json');
+                //this is used specifically for excel imports 'from web' via Report Builder toolbar -> JSON -> compat mode
+                if($format === 'jsoncompatmode') {
```

Comment:
I know we had chatted on this, was there no way to determine if this was for the excel importer already? Just so people do not have to pull it in a different way or reimport the file. I am not sure how easy it is to just swap out a datasource in those tools.

Reply:
I will touch in on this Monday.  I'd actually prefer to have it in the Shortener's getFormQuery for a few reasons.